components/kyma-environment-broker/internal/orchestration/handlers/kyma_handler.go

```diff
@@ -206,3 +210,28 @@ func (h *kymaHandler) resolveErrorStatus(err error) int {
 		return http.StatusInternalServerError
 	}
 }
+
+func (h *kymaHandler) defaultTarget(spec *internal.TargetSpec) {
```

Comment:
I suggest we make at least one include target mandatory. Defaulting to target all can lead to major accidents.